dart/lib/src/sentry_exception_factory.dart

```diff
@@ -45,13 +45,11 @@ class SentryExceptionFactory {
 
     // if --obfuscate feature is enabled, 'type' won't be human readable.
     // https://flutter.dev/docs/deployment/obfuscate#caveat
-    final sentryException = SentryException(
-      type: '${throwable.runtimeType}',
-      value: '$throwable',
+    return SentryException(
+      type: (throwable.runtimeType).toString(),
+      value: throwable.toString(),
```

Comment:
Just out of curiosity, is `.toString` preferred over String interpolation?

Reply:
I would say so, if it's just a single variable. Though, I can revert it, if you want me to.

Reply:
No, it actually makes sense, I was just curious if there's some sort of linting for that.

Reply:
Actually there is https://dart-lang.github.io/linter/lints/unnecessary_string_interpolations.html